.github/workflows/Docker.yaml

```diff
@@ -1,29 +0,0 @@
-name: Build and push a new docker image based on project release tag.
```

Comment:
Why did this file got deleted?

Reply:
It was deleted by mistake. I've restored it now.